### PR TITLE
RNMT-5913 - Update SQLite

### PR DESCRIPTION
**Pull request by Chuckytuh**
This PR introduces the upstream changes that most importantly include:
- Recent changes from https://github.com/brodybits/android-sqlite-ndk-native-driver/commit/f6e41f6ee546002a388e9404d916c86711e184c4 
  - That repository contains the SQLite version bump to 3.32.3
- Recent changes from https://github.com/brodybits/android-sqlite-native-ndk-connector/commit/d5d72f8a2b5a23cf4344d8721c98d89e1a952e8d
  - That repository contains the Java API to abstract away from the JNI API that communicates with the C counterpart brought in by `android-sqlite-ndk-native-driver`

These changes are introduced via npm package https://www.npmjs.com/package/cordova-sqlite-storage-dependencies (https://github.com/brodybits/cordova-sqlite-storage-dependencies/commit/a57b0a1b30dfd7e83e794aa208e137f7b5c29193)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,11 +1,12 @@
 {
   "name": "cordova-sqlite-storage",
-  "version": "3.2.0-OS3",
-  "description": "Native interface to SQLite for PhoneGap/Cordova - cordova-sqlite-storage plugin version",
+  "version": "6.0.0-OS1",
```
Comment (EiyuuZack) [inline package.json:3]: According to the new specification this should not use the `-OSX` suffix but instead use `+<semver>` suffix. Can be fixed here or in a posterior raise commit (which will be tagged).

File: plugin.xml [hunk 2]
```diff
@@ -2,9 +2,9 @@
 <plugin xmlns="http://www.phonegap.com/ns/plugins/1.0"
     xmlns:android="http://schemas.android.com/apk/res/android"
     id="cordova-sqlite-storage"
-    version="3.2.0-OS3">
+    version="6.0.0-OS1">
```
Comment (EiyuuZack) [inline plugin.xml:5]: According to the new specification this should not use the `-OSX` suffix but instead use `+<semver>` suffix. Can be fixed here or in a posterior raise commit (which will be tagged).

**Review by EiyuuZack — APPROVED** [review]

File: plugin.xml (hunk 2 above)
Comment (Chuckytuh) [reply plugin.xml:5]: Thanks for bringing this up, I was actually going to raise these versions on a raise commit but this was an oversight, I missed it. I wanted this to have the original, upstream, version to make it explicit from where we're basing our version

**Review by tiagoappereira — APPROVED** [review]
LGTM

**Pull request merged**